tests/hypertext/HTMLLintBearTest.py

```diff
@@ -27,3 +27,10 @@
     invalid_files=(test_file,),
     settings={'htmllint_ignore': 'quotation'},
     tempfile_kwargs={'suffix': '.html'})
+
+HTMLLintBearDisableCheckerTest = verify_local_bear(
+    HTMLLintBear,
+    valid_files=(test_file,),
+    invalid_files=(),
```

Comment:
The tests before don't follow that. The `invalid_files` is already covered in the `HTMLLintBearTest` and `HTMLLintBearIgnoreQuotationTest`. I feel since `HTMLLintBearDisableCheckerTest` is supposed to check whether disabling a certain setting takes effect, it would make no sense to include an invalid_file.